src/engine/sidechain/enginerecord.cpp

```diff
@@ -249,6 +251,12 @@ void EngineRecord::writeCueLine() {
                             .arg(m_pCurrentTrack->getArtist())
                             .toUtf8());
 
+    if (m_bCueUsesFileAnnotation) {
+        m_cueFile.write(QString("    FILE \"%1\"\n")
```

Comment:
```suggestion
        m_cueFile.write(QStringLiteral("    FILE \"%1\"\n")
```

Feel free to also fix other occurrence

Reply:
Just pushed new commit that should take care of this. I updated the line you suggested as well as the ones above it. Thanks!